Simple Calculator/src/Calculator.java

```diff
@@ -0,0 +1,60 @@
+import java.util.Scanner;
+
+public class Calculator {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter the first number");
+
+        if (sc.hasNextDouble()) {
+            double firstValue = sc.nextDouble();
+            System.out.println("Input is correct: " + firstValue);
+
+            System.out.println("Enter an operation (for example: +, -, *, /, %)");
+            String operation = sc.next();
+            System.out.println(operation);
+            if (operation.equals("+") |
+                    operation.equals("-") |
+                    operation.equals("*") |
+                    operation.equals("/") |
+                    operation.equals("%")) {
+                System.out.println("User inputs supported operations");
+
+                System.out.println("Enter the second number");
+                if (sc.hasNextDouble()) {
+                    double secondValue = sc.nextDouble();
+                    System.out.println("Input is correct: " + secondValue);
+                    if (operation.equals("+")) {
+                        double result = firstValue + secondValue;
```

Comment:
Please initialize variable at the beginning of method, and here and below just use it

Reply:
done